### PR TITLE
Set up PostgreSQL Connection

### PR DESCRIPTION
**Pull request by geofflin**
- Installed dependencies (dotenv, pg) in package.json
 - Created **db** folder at root level directory
 - Created **psqlPool.js** file to connect to PostgreSQL database for both production and development based on node environment variable mode

### CODE REVIEW

File: db/psqlPool.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  pool = new Pool({
+    host: 'localhost',
+    database: process.env.DEV_DB,
+    user: process.env.DEV_DB_USR,
+    password: process.env.DEV_DB_PW,
+  });
+}
+
+pool.connect((err, client, done) => {
+  if (err) return console.error('Unsuccessfully connected to db');
```
Comment (Donaldblanc) [inline db/psqlPool.js]: Can you console log out the error as well ?

File: db/psqlPool.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  pool = new Pool({
+    user: process.env.PROD_DB_USR,
+    host: process.env.PROD_HOST,
+    database: process.env.PROD_DB,
+    password: process.env.PROD_DB_PW,
+    ssl: true,
+    port: 5432,
+  });
+} else {
+  console.log(process.env.DEV_DB_PW);
```
Comment (Donaldblanc) [inline db/psqlPool.js]: Lets remove the console logs that exposes the environment. Not great practice.

File: package.json [hunk 3]
```diff
@@ -21,5 +21,9 @@
   "bugs": {
     "url": "https://github.com/cipherchase/cipherchase/issues"
   },
-  "homepage": "https://github.com/cipherchase/cipherchase#readme"
+  "homepage": "https://github.com/cipherchase/cipherchase#readme",
+  "dependencies": {
```
Comment (Donaldblanc) [inline package.json:25]: dotenv should be a Dev dependency, lets move it there.

**Review by Donaldblanc — CHANGES_REQUESTED** [review]
Overall great Job Geoff!  lets make these changes and we are good to go.

File: db/psqlPool.js (hunk 1 above)
Comment (geofflin) [reply db/psqlPool.js]: Resolved: console logged the error.
Also removed extraneous console log

File: db/psqlPool.js (hunk 2 above)
Comment (geofflin) [reply db/psqlPool.js]: Resolved: Removed console logs that exposed environment

File: package.json (hunk 3 above)
Comment (geofflin) [reply package.json:25]: dotenv moved to devDependency

**Review by jaeroong — APPROVED** [review]
looks good

**Review by Donaldblanc — APPROVED** [review]
👍 thanks for makes those changes. this looks great!

**Pull request merged**